Code/Framework/AzCore/AzCore/Math/Vector2.h

```diff
@@ -163,6 +163,18 @@ namespace AZ
         //! The positive rotation direction is defined such that the x-axis is rotated into the y-axis.
         Vector2 GetPerpendicular() const;
 
+        // carbonated begin aoreshko game_specific_1
```

Comment:
According to the document we skip names here, just the feature name. We keep names in a document registry only.
Please remove aoreshko,